apps/condo/domains/organization/integrations/sbbol/routes.js

```diff
@@ -21,7 +22,8 @@ class SbbolRoutes {
         // nonce: to prevent several callbacks from same request
         // state: to validate user browser on callback
         const checks = { nonce: generators.nonce(), state: generators.state() }
-        req.session[SBBOL_SESSION_KEY] = checks
+        const redirectUrl = get(req, 'query.redirectUrl')
+        req.session[SBBOL_SESSION_KEY] = { checks, redirectUrl: isSafeUrl(redirectUrl) ? redirectUrl : undefined }
```

Comment:
It's better to throw exception if `redirectUrl` is incorrect.
Otherwise, user will have no clue that something is wrong.